pkg/plugins/builtin/ssh/README.md

```diff
@@ -42,15 +42,28 @@ action:
     ssh_key: '{{.config.mySSHKey}}'
 ```
 
-The output resulting from this configuration will be:
+## Requirements
+
+None by default. Ssh credentials should be retrieved from `{{.config.[sshkey]}}` rather than hardcoded on the template.
+
+## Note
+
+The plugin returns two objects, the `Payload` of the execution:
```

Comment:
It's worth noting that the format of this payload is set in the **result** field of the step configuration

Reply:
fixed.